File: src/difficulty_1/difficulty_1_3/pot/Pot.java
package difficulty_1.difficulty_1_3.pot;

import java.util.Scanner;

public class Pot {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int count = scanner.nextInt();

        int x = 0;
        for(int i=0; i<count; i++) {
            int p = scanner.nextInt();
            x += Math.pow(p/10, p%10);
        }
        scanner.close();

        System.out.println(x);
    }

}

File: KotlinImplementation/src/difficulty_1/Pot.kt
package difficulty_1

import kotlin.math.pow

fun main() {
    val N = readLine()!!.toInt()
    var X = 0.0

    for (i in 1..N) {
        val nr = readLine()!!.toInt()
        X += Math.pow(nr / 10 * 1.0, nr % 10 * 1.0)
    }

    println(X.toInt())
}